File: helixtrajectoryj/src/main/java/org/team2363/helixtrajectory/HolonomicDrivetrain.java
package org.team2363.helixtrajectory;

public abstract class HolonomicDrivetrain extends Drivetrain {
    
    protected HolonomicDrivetrain(double mass, double momentOfInertia, Obstacle bumpers) throws NullPointerException {
        super(mass, momentOfInertia, bumpers);
    }
}

File: helixtrajectorycpp/src/HolonomicDrivetrain.cpp
#include "HolonomicDrivetrain.h"

#include "Drivetrain.h"
#include "Obstacle.h"

namespace helixtrajectory {

    HolonomicDrivetrain::HolonomicDrivetrain(double mass, double momentOfInertia, const Obstacle& bumpers)
        : Drivetrain(mass, momentOfInertia, bumpers) {
    }

    HolonomicDrivetrain::~HolonomicDrivetrain() {
    }
}